modAion/src/org/aion/zero/types/AionTransaction.java

```diff
@@ -251,7 +251,7 @@ public Address getDestinationAddress() {
     }
 
     @Override
-    public byte getTransactionType() {
+    public byte getTargetVM() {
```

Comment:
Can we reverse this renaming? After some discussion, it was concluded that it is too restrictive and we should keep the option of multiple transaction types.

Reply:
We will reverse this on our next vm api iteration. Just because we have already tagged the build and made a release in the vm_api repository so that the AVM could have a reliable version history, so it's quite a bit of work to undo it right now

Reply:
ok